components/NewFooter/__tests__/index.test.tsx

```diff
@@ -0,0 +1,26 @@
+import { render } from '@testing-library/react';
+import { LocaleProvider } from '../../../providers/localeProvider';
+import Footer from '..';
+import type { AppProps } from '../../../types';
+
+// mock useRouter
+jest.mock('next/router', () => ({
+  useRouter() {
+    return {
+      locale: 'en',
+    };
+  },
+}));
+
+const i18nData = { currentLocale: { code: 'en' } } as AppProps['i18nData'];
+
+describe('Tests for Footer component', () => {
+  it('renders correctly', () => {
+    const { container } = render(
+      <LocaleProvider i18nData={i18nData}>
```

Comment:
Again, we should be using `IntlProvider` without actual i18n data on unit tests :)

Reply:
Not in this case be cause we need intProvider for `IntlLink`

Reply:
WDYM? LocaleProvider is a wrapper of IntlProvider. Please just use `IntlProvider`. `IntlProvider` defaults to "locale" `en`

Reply:
IDK why (I'm searching on that) but the unit test fails if I use `LocaleProvider`